### PR TITLE
45-workforce-db :: Connecting DB to workforce management database

### PR DESCRIPTION
**Pull request by matthamil**
## Status
**DEPENDENT UPON https://github.com/TeamCharles/workforce-management/pull/85**

## Testing

[ ] I have added passing tests for all new functionality.

[X] I have confirmed that all test prior to this pull request still PASS.

## Documentation

[ ] I have fully documented features in this PR's code. Please put an `X` in the box to confirm.

[X] This PR does **NOT** require documentation.

## Description
Workforce Management database now plugged in to this app.

## Resolves Issue Number

https://github.com/TeamCharles/customer-service/issues/45

## Deploy Notes

```sh
git pull --prune
git checkout 45-workforce-db
dotnet run
```

## Steps to Test or Reproduce

1. Employees and departments now come from old DB.

## Impacted Files in Application

* Factories/EmployeeFactory.cs

* Factories/DepartmentFactory.cs

* Models/Employee.cs

### CODE REVIEW

**Comment by daynewright** [comment]
LGTM

**Comment by daynewright** [comment]
LGTM 👍

File: README.md [hunk 1]
```diff
@@ -24,15 +24,21 @@ export ASPNETCORE_ENVIRONMENT="Development"
...
 On initial installation of the Banagazon CLI application you must set an environment variable to your local database.
 ```Bash
-export BangazonWeb_Db_Path="/path/to/bangazon.db"
+export Bangazon_Incident_Db="/path/to/bangazon_incident.db"
 ```
 
-Open your `bangazon.db` file in DB Browser, select the `Edit Pragmas` tab and make sure that the `Foreign Keys` checkbox is checked. Open `Data/tables.sql` from the `Execute SQL` tab of DB Browser and execute the statement. Do the same action with `Data/populate.sql`. In DB Browser, click `Write Changes` to commit your changes to the database.
+This project depends upon the database files created by two other projects.
+
+1. Clone or download Team Charles's [Initial Site](https://github.com/TeamCharles/initial-site) and follow the instructions in the README.
```
Comment (MattKraatz) [inline README.md:37]: Documentation is lit

File: src/customer-service/Models/Employee.cs [hunk 2]
```diff
@@ -17,6 +17,11 @@ namespace customer_service.Models
 
     public class Employee
     {
+        public Employee()
+        {
+            StartDate = DateTime.Now;
```
Comment (MattKraatz) [inline src/customer-service/Models/Employee.cs:22]: noice

**Review by MattKraatz — APPROVED** [review]
LGTM

**Comment by garrett-vangilder** [comment]
LGTM

**Pull request merged**